Update-UserScript.py

```diff
@@ -0,0 +1,81 @@
+##############################################################################
+# (C) Copyright [2020] Hewlett Packard Enterprise Development LP
+#
+# File Name: Update-UserScript.py
+# VERSION 1.0
+# Usage: python Update-UserScript.py <SDK_REPO_PATH>
+#
+# This script can be used independently from the hpeOneView 5.4.0 library to
+# parse a user script and update all legacy hpeOneView brand names from
+# hpOneView to hpeOneView, and any reference to HPOneView to HPEOneView.
+#
+##############################################################################
+# 
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+# 
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+# 
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+# 
+##############################################################################
+
+import re
+import os
+import sys
+
+self_name = sys.argv[0]
+path = sys.argv[1] # Fetch the SDK repository path from cmd line
+search_pattern1 = re.compile('hpOneView')
+search_string1 = 'hpOneView'
+replace_string1 = 'hpeOneView'
```

Comment:
Why can't you have it as HPEOneView in both "hpOneView" and "hpeOneView"?

Reply:
few class names in python sdk are defined as HPEOneView and python module is defined as hpeOneView. Since there is a mix of "hpe" and "HPE", I am searching and replacing both the strings seperately.